### PR TITLE
[Merged by Bors] - Initialize open telemetry

### PR DESCRIPTION
**Pull request by nacardin**

### CODE REVIEW

File: rust-connectors/sinks/kafka/src/main.rs [hunk 1]
```diff
@@ -10,6 +10,7 @@ use tokio_stream::StreamExt;
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
+    init_open_telemetry();
```
Comment (sehz) [inline rust-connectors/sinks/kafka/src/main.rs:13]: Move this under like `common_initialize` so we can consolidate into a single initialization.  In fact, we should have common connector `main` like `fluvio future` (let's take as another issue for that).

**Review by sehz — CHANGES_REQUESTED** [review]
A minor comment

File: rust-connectors/sinks/kafka/src/main.rs (hunk 1 above)
Comment (nacardin) [reply rust-connectors/sinks/kafka/src/main.rs]: done

**Review by sehz — APPROVED** [review]
LGTM.

**Comment by nacardin** [comment]
bors retry

**Pull request closed without merging**